applications/workspaces/server/workspaces/service/osbrepository/adapters/githubadapter.py

```diff
@@ -85,8 +96,19 @@ def get_description(self, context):
         except Exception as e:
             return e
```

Comment:
This is something I missed before: Why return an exception?

Reply:
okay, I'll return ""